app/src/main/java/com/codesoom/assignment/model/Task.java

```diff
@@ -0,0 +1,68 @@
+package com.codesoom.assignment.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+/**
+ * 사용자가 등록한 할 일을 나타냅니다.
```

Comment:
사용자가 등록하지 않은 할 일이 생길 수도 있겠지요 :-) 이렇게 해보면 어떨까요?
```suggestion
 * 할 일.
```

Reply:
사용자가 등록하지 않은 할 일이 뭔지 오래 생각해봤는데...다른 서비스가 자동으로 등록해줄 수도 있겠네요.

혹시 JavaDoc을 작성하는 기준은 어떻게 정하시는 편이신가요?

> 저는 '~다.'로 끝나는 절의 형식이어야 한다고 고정관념이 있었던 것 같아요. 스프링 JavaDoc도 보니까 첫 문장은 구 형식이네요.

Reply:
명사형의 개념으로 표현할 것인지, 책임을 강조하는 문장형으로 작성할 것인지를 고민하고 정하는 편이에요.
JavaDoc을 빌드했을 때 html 결과가 어떤 식으로 나오게 될 지도 염두에 둡니다.